### ISSUE TITLE
Is that possible to be a slightly different with DSSAT itself?

### ISSUE THREAD

**Issue by danialll1** [daquinterop/Py_DSSATTools#38]
Hi there
After creating an example in both DSSAT and DSSATTools I'v noticed with the totally same setting could receive different values in two environment and was wondering is that possible or I'v done something wrong?
I'v checked the situations like 100times and they are exactly the same.
P.S: Maize was the crop and if you need to have the files let me know!
Best

**Comment by daquinterop** [comment]
Yep, they can be different,. It can happen due to the way the library formats the input files for DSSAT. For example, decimal places can make difference in that case. How different are they? If they are completely different then it must be something wrong with the library.

**Comment by danialll1** [comment]
Hi again
I provide a genetic file which is Maize and the Soil which could be the reason of problem 
because when I run them in DSSATTools and DSSAT could receive off answer like %90 and in least situation I recieved something about 2 percent off in relation to DSSAT solution.
[DSSAT_TOOLS.zip](https://github.com/user-attachments/files/16736411/DSSAT_TOOLS.zip)
best

**Comment by daquinterop** [comment]
Hi @danialll1  Thanks for your detailed answer. That sounds like an important problem. I see that you shared SOL and CUL files, Could you please share all the files that you find to have different results, including .WTH and MZX files?

**Comment by danialll1** [comment]
Hi dear @daquinterop 
thanks for your response my friend
Files that You asked have been attached. 
[Archive.zip](https://github.com/user-attachments/files/16739952/Archive.zip)
hope the problem will fix.
Best

**Comment by daquinterop** [comment]
That MZX file includes 14 treatments. Can you please tell me one that generates different results for me to focus on that one? Also The cultivar and soils you shared are not referenced in that MZX.
